pkg/provider/alibaba/kms.go

```diff
@@ -140,16 +140,23 @@ func (kms *KeyManagementService) GetSecret(ctx context.Context, ref esv1alpha1.E
 	return []byte(val.String()), nil
 }
 
+// Implements store.Client.GetAllSecrets Interface.
+// New version of GetAllSecrets.
+func (kms *KeyManagementService) GetAllSecrets(ctx context.Context, ref esv1alpha1.ExternalSecretDataFromRemoteRef) (map[string][]byte, error) {
+	// TO be implemented
```

Comment:
Wouldn't it be better to return "Not Implemented" Error? Otherwise for the user this will be a silent fail (External Secrets in "SecretSynced", but with no content)